pkg/sql/pg_catalog.go

```diff
@@ -2775,6 +2774,52 @@ CREATE TABLE pg_catalog.pg_type (
 				})
 			})
 	},
+	indexes: []virtualIndex{
+		{
+			partial: false,
+			populate: func(ctx context.Context, constraint tree.Datum, p *planner, db *sqlbase.ImmutableDatabaseDescriptor,
+				addRow func(...tree.Datum) error) (bool, error) {
+
+				h := makeOidHasher()
+				nspOid := h.NamespaceOid(db, pgCatalogName)
+				coid := tree.MustBeDOid(constraint)
+				ooid := oid.Oid(int(coid.DInt))
+
+				// Check if it is a predefined type.
+				typ, ok := types.OidToType[ooid]
+				if ok {
+					if err := addPGTypeRow(h, nspOid, typ, addRow); err != nil {
+						return false, err
+					}
+					return true, nil
+				}
+
+				// Check if it is a user defined type.
+				id := sqlbase.ID(types.UserDefinedTypeOIDToID(ooid))
+				// TODO (rohany): This access should go through the desc.Collection.
+				typDesc, err := sqlbase.GetTypeDescFromID(ctx, p.txn, keys.SystemSQLCodec, id)
+				if err != nil {
+					if err == sqlbase.ErrDescriptorNotFound {
```

Comment:
use `errors.Is` here rather than a direct comparison

Reply:
Done.